File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/AllyInfernalDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

public class AllyInfernalDragonKillAnimation extends StaticBlinkingAnimation {
    public AllyInfernalDragonKillAnimation() {
        super(8, DragonType.INFERNAL.getColor());
    }
}

File: src/hud/animations/AllyInfernalDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {AllyDragonKillBlinkingAnimation} from './AllyDragonKillBlinkingAnimation.js';

export class AllyInfernalDragonKillAnimation extends AllyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.INFERNAL);
    }
}
